internal/provisioner/plain/README.md

```diff
@@ -4,34 +4,13 @@
 
 The `plain` provisioner is a core rukpak provisioner that knows how to interact with bundles of a particular format.
 These `plain+v0` bundles, or plain bundles, are simply container images containing a set of static Kubernetes YAML
-manifests in a given directory. For example, below is an example Dockerfile that builds a plain `plain+v0` bundle from a
-directory containing static manifests.
+manifests in a given directory. For more information on the `plain+v0` format, see 
+the [plain+v0 bundle spec](/docs/plain-bundle-spec.md).
 
-```dockerfile
-FROM scratch
-COPY /manifests /manifests
-```
-
-where the given `manifests` directory contains the Kubernetes resources required to deploy an application, for example:
-
-```
-manifests
-├── namespace.yaml
-├── cluster_role.yaml
-├── role.yaml
-├── serviceaccount.yaml
-├── cluster_role_binding.yaml
-├── role_binding.yaml
-└── deployment.yaml
-```
-
-> Note: The static manifests must be located in the root-level /manifests directory for the bundle image to be a
-valid `plain+v0` bundle that the provisioner can unpack.
-
-The `plain` provisioner is able to unpack a given `plain+v0` bundle onto a cluster and then instantiate it, making
-the content of the bundle available in the cluster. It does so by reconciling `Bundle` and `BundleInstance` types that
-have the `spec.provisionerClassName` field set to `core.rukpak.io/plain`. This field must be set to the correct
-provisioner name in order for the `plain` provisioner to see and interact with the bundle.
+The `plain` provisioner is able to unpack a given `plain+v0` bundle onto a cluster and then instantiate it, making the
```

Comment:
The changes here and below in this file are all as a result of markdown linting.